tests/page-objects/ql/QLPage.ts

```diff
@@ -143,6 +156,74 @@ class QLPage extends ChartPage {
         await this.saveAsNewChart(entryName);
     }
 
+    async openParamsTab() {
+        await this.page.click(
+            `.ql-screen-editor .gc-adaptive-tabs__tab-container_last-tab .gc-adaptive-tabs__tab`,
+        );
+    }
+
+    async addParam() {
+        await this.page.click(slct(QLPageElementsQA.AddParamBtn));
+    }
+
+    async selectParamType(type: string) {
+        await this.page.click(slct(QLPageElementsQA.ParamType));
+
+        await this.page.click(`${CommonSelectors.SelectItem} >> text=${type}`);
+    }
+
+    async setParamName(name: string) {
+        const textInput = await this.page.waitForSelector(
```

Comment:
it is better to use locators (waitForSelector is deprecated)
https://playwright.dev/docs/api/class-page#page-wait-for-selector